broker/src/main/java/io/zeebe/broker/bootstrap/StartProcess.java

```diff
@@ -90,7 +94,7 @@ static long takeDuration(final CheckedRunnable runner) throws Exception {
     return endTime - startTime;
   }
 
-  private static class StartStep {
+  public static class StartStep {
```

Comment:
Why is this necessary?

Reply:
No, this is a useless change, I'll revert this